libmscore/arpeggio.cpp

```diff
@@ -22,506 +22,524 @@
-
 const std::array<const char*, 6> Arpeggio::arpeggioTypeNames = {
-      QT_TRANSLATE_NOOP("Palette", "Arpeggio"),
-      QT_TRANSLATE_NOOP("Palette", "Up arpeggio"),
-      QT_TRANSLATE_NOOP("Palette", "Down arpeggio"),
-      QT_TRANSLATE_NOOP("Palette", "Bracket arpeggio"),
-      QT_TRANSLATE_NOOP("Palette", "Up arpeggio straight"),
-      QT_TRANSLATE_NOOP("Palette", "Down arpeggio straight")
-      };
-
-//---------------------------------------------------------
-//   Arpeggio
-//---------------------------------------------------------
-
-Arpeggio::Arpeggio(Score* s)
-  : Element(s, ElementFlag::MOVABLE)
-      {
-      _arpeggioType = ArpeggioType::NORMAL;
-      setHeight(spatium() * 4);      // for use in palettes
-      _span     = 1;
-      _userLen1 = 0.0;
-      _userLen2 = 0.0;
-      _playArpeggio = true;
-      _stretch = 1.0;
-      }
-
-//---------------------------------------------------------
-//   setHeight
-//---------------------------------------------------------
+    QT_TRANSLATE_NOOP("Palette", "Arpeggio"),
+    QT_TRANSLATE_NOOP("Palette", "Up arpeggio"),
+    QT_TRANSLATE_NOOP("Palette", "Down arpeggio"),
+    QT_TRANSLATE_NOOP("Palette", "Bracket arpeggio"),
+    QT_TRANSLATE_NOOP("Palette", "Up arpeggio straight"),
+    QT_TRANSLATE_NOOP("Palette", "Down arpeggio straight")
+};
+
+// ---------------------------------------------------------
+// Arpeggio
+// ---------------------------------------------------------
+
+Arpeggio::Arpeggio(Score* s) :
+    Element(s, ElementFlag::MOVABLE)
+{
+    _arpeggioType = ArpeggioType::NORMAL;
+    setHeight(spatium() * 4);        // for use in palettes
+    _span = 1;
+    _userLen1 = 0.0;
+    _userLen2 = 0.0;
+    _playArpeggio = true;
+    _stretch = 1.0;
+}
+
+// ---------------------------------------------------------
+// setHeight
+// ---------------------------------------------------------
 
 void Arpeggio::setHeight(qreal h)
-      {
-      _height = h;
-      }
+{
+    _height = h;
+}
 
-//---------------------------------------------------------
-//   write
-//---------------------------------------------------------
+// ---------------------------------------------------------
+// write
+// ---------------------------------------------------------
 
 void Arpeggio::write(XmlWriter& xml) const
-      {
-      if (!xml.canWrite(this))
-            return;
-      xml.stag(this);
-      Element::writeProperties(xml);
-      writeProperty(xml, Pid::ARPEGGIO_TYPE);
-      if (_userLen1 != 0.0)
-            xml.tag("userLen1", _userLen1 / spatium());
-      if (_userLen2 != 0.0)
-            xml.tag("userLen2", _userLen2 / spatium());
-      if (_span != 1)
-            xml.tag("span", _span);
-      writeProperty(xml, Pid::PLAY);
-      writeProperty(xml, Pid::TIME_STRETCH);
-      xml.etag();
-      }
-
-//---------------------------------------------------------
-//   read
-//---------------------------------------------------------
+{
+    if (!xml.canWrite(this))
```

Comment:
uncrustify has "mod_full_brace_if (ON / OFF)"
I think that we can force brackets for single-line statements in the config

Reply:
fixed